lightningd/opening_control.c

```diff
@@ -1052,13 +1052,20 @@ static struct command_result *json_fund_channel_continue(struct command *cmd,
 	struct bitcoin_txid *funding_txid;
 	struct peer *peer;
 	struct channel *channel;
+	u32 *funding_txout_num;
+	u16 funding_txout;
 
 	if (!param(cmd, buffer, params,
 		   p_req("id", param_node_id, &id),
 		   p_req("txid", param_txid, &funding_txid),
+		   p_req("txout", param_number, &funding_txout_num),
 		   NULL))
 		return command_param_failed();
 
+	if (*funding_txout_num > UINT16_MAX)
+		return command_param_failed();
```

Comment:
command_param_failed is kinda magical: you need `command_fail()` here with a message.